src/Dfc.CourseDirectory.WebV2/Models/ApprenticeshipQAUnableToCompleteReasons.cs

```diff
@@ -12,4 +12,28 @@ public enum ApprenticeshipQAUnableToCompleteReasons
         ProviderHasAppliedToTheWrongRoute = 8,
         Other = 16
     }
+
+    public static class ApprenticeshipQAUnableToCompleteReasonsExtensions
+    {
+        public static string ToDisplayName(this ApprenticeshipQAUnableToCompleteReasons status)
+        {
+            switch (status)
+            {
+                case ApprenticeshipQAUnableToCompleteReasons.None:
+                    return "None";
+                case ApprenticeshipQAUnableToCompleteReasons.Other:
+                    return "Other";
+                case ApprenticeshipQAUnableToCompleteReasons.ProviderDevelopingProvision:
+                    return "Provider Developing Provision";
+                case ApprenticeshipQAUnableToCompleteReasons.ProviderHasAppliedToTheWrongRoute:
+                    return "Provider has applied to the wrong route";
+                case ApprenticeshipQAUnableToCompleteReasons.ProviderHasWithdrawnApplication:
+                    return "Provider has withdrawn application";
+                case ApprenticeshipQAUnableToCompleteReasons.StandardNotReady:
+                    return "Standard not ready";
+                default:
+                    throw new NotImplementedException($"Unknown value: '{status}'.");
+            }
+        }
+    }
```

Comment:
nitpicky, but you could add a DescriptionAttribute to each enum to store the display text

Reply:
@soharadesign there are places where this 'display name' logic is more complex than a straight one to one translation so DescriptionAttribute won't work there and I'd like to keep consistent approach where we can